qiskit/compiler/disassembler.py

```diff
@@ -41,8 +42,11 @@ def _experiments_to_circuits(qobj):
             for reg in classical_registers:
                 creg_dict[reg.name] = reg
             for i in x.instructions:
-                instr_method = getattr(circuit, i.name)
+                name = i.name
+                if i.name == 'id':
+                    name = 'iden'
                 qubits = []
+                params = []
```

Comment:
Can we avoid the `broad-except` for params with something like `params = getattr(i, 'params', [])`?

Reply:
Yeah, that should work fine too. I'm not sure why I did it this way, probably just to minimize the patch delta

Reply:
Since I had to manually rebase anyway I took the opportunity to update this too in: https://github.com/Qiskit/qiskit-terra/pull/2147/commits/dd429928ec64d6279e0647b754d138d60d9ac86a